src/libraries/System.Text.Json/src/System/Text/Json/ThrowHelper.cs

```diff
@@ -87,6 +87,12 @@ public static void ThrowInvalidOperationException_NeedLargerSpan()
             throw GetInvalidOperationException(SR.FailedToGetLargerSpan);
         }
 
+        [DoesNotReturn]
+        public static void ThrowPropertyNameTooLargeArgumentException(int length)
+        {
+            throw GetArgumentException(SR.Format(SR.PropertyNameTooLarge, length));
```

Comment:
If the `PropertyNameTooLarge` res string already exists, shouldn't there be a corresponding throw method already? Or was it unused/used differently until now?

Reply:
PropertyNameTooLarge is used in other throw helpers that check multiple arguments, one of which is a property being checked for its length.